### PR TITLE
fix: fix varbinary type error

### LINKED ISSUE #637: Unsupported SQL data type, err:Unsupported SQL data type, type:VARBINARY.
### Describe this problem

Create table with varbinary will throw error.
```
2023-02-10 18:04:16.977 ERRO [server/src/grpc/storage_service/mod.rs:257] Failed to handle request, mod:sql_query, handler:handle_query, err:Rpc error, code:500, message:Failed to create plan, qu
ery:CREATE TABLE IF NOT EXISTS ceresdb (                                                                                                                                                           
                str_tag string TAG,                                                                                                                                                                
                int_tag int32 TAG,                                                                                                                                                                 
                var_tag VARBINARY TAG,                                                                                                                                                             
                str_field string,                                                                                                                                                                  
                int_field int32,                                                                                                                                                                   
                bin_field string,                                                                                                                                                                  
                t timestamp NOT NULL,                                                                                                                                                              
                TIMESTAMP KEY(t)) ENGINE=Analytic with                                                                                                                                             
(enable_ttl='false'), cause:Failed to create plan, err:Unsupported SQL data type, err:Unsupported SQL data type, type:VARBINARY.                                                                   
Backtrace:                                                                                       
 0 <snafu::backtrace_shim::Backtrace as snafu::GenerateBacktrace>::generate::hae5493024171cea8                                                                                                     
   /home/chenxiang/.cargo/registry/src/github.com-1ecc6299db9ec823/snafu-0.6.10/src/backtrace_shim.rs:15
   common_types::datum::UnsupportedDataType<__T0>::build::hd328396a801bd9d0                                                                                                                        
   /home/chenxiang/ceresdb/common_types/src/datum.rs:14                                     
 1 common_types::datum::UnsupportedDataType<__T0>::fail::hf24c888d994be662                                                                                                                         
   /home/chenxiang/ceresdb/common_types/src/datum.rs:14                             
 2 <common_types::datum::DatumKind as core::convert::TryFrom<&sqlparser::ast::data_type::DataType>>::try_from::h52db0aa41eec82d4
   /home/chenxiang/ceresdb/common_types/src/datum.rs:235                                                                                                                                           
 3 sql::planner::parse_column::h61a0a174a1cff1ca                                                                                                                                                   
   /home/chenxiang/ceresdb/sql/src/planner.rs:951                                                                                                                                                  
 4 sql::planner::PlannerDelegate<P>::create_table_to_plan::{{closure}}::h0cc4f7e8384aa3d1                                                                                                          
   /home/chenxiang/ceresdb/sql/src/planner.rs:478                                                                                                                                                  
 5 core::iter::adapters::map::map_try_fold::{{closure}}::h4b9fbdbaf06092f6                                                                                                                         
   /rustc/d394408fb38c4de61f765a3ed5189d2731a1da91/library/core/src/iter/adapters/map.rs:91                                                                                                        
 6 core::iter::traits::iterator::Iterator::try_fold::hd566d40d047a9498 
```

### Steps to reproduce

See create table statement above

### Expected behavior

No error

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by ZuLiangWang**
# Which issue does this PR close?

Closes #637 

# Rationale for this change
Fix the problem of table creation failure mentioned in issue.

# What changes are included in this PR?
* Fix `SqlDataType` convert error.
* Add create table test with `VARBINARY` type. 

# Are there any user-facing changes?
User could create table with `VARBINARY` type normally.

# How does this change test
Pass integration tests.

### CODE REVIEW

File: common_types/src/datum.rs [hunk 1]
```diff
@@ -213,6 +213,7 @@ impl TryFrom<&SqlDataType> for DatumKind {
             SqlDataType::Int(_) => Ok(Self::Int32),
             SqlDataType::SmallInt(_) => Ok(Self::Int16),
             SqlDataType::String => Ok(Self::String),
+            SqlDataType::Varbinary(_) => Ok(Self::Varbinary),
             SqlDataType::Custom(objects, _) if objects.0.len() == 1 => {
                 match objects.0[0].value.as_str() {
                     "UINT64" | "uint64" => Ok(Self::UInt64),
```
Comment (ShiKaiWi) [inline common_types/src/datum.rs:219]: Shall we add some tests for these types?

Comment (ZuLiangWang) [reply common_types/src/datum.rs:219]: Good suggestion. I will add a test case that includes all types.

File: integration_tests/cases/local/03_dml/issue-637.result [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+affected_rows: 0
+
+CREATE TABLE IF NOT EXISTS `issue637`
+(
+    str_tag string TAG,
+    int_tag int32 TAG,
+    var_tag VARBINARY TAG,
+    str_field string,
+    int_field int32,
```
Comment (Rachelint) [inline integration_tests/cases/local/03_dml/issue-637.result:11]: How about testing all data types?

**Review by ShiKaiWi — APPROVED** [review]
LGTM

**Pull request merged**